docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+
+Many TestCafe users take advantage of Jenkins — a CI/CD solution — to automate their tests. Until recently, they had to manually match the screenshots and videos taken by TestCafe to the individual test reports. The newly implemented [TestCafe Jenkins plugin](https://plugins.jenkins.io/testcafe/) simplifies this process. Links to screenshots and videos taken during the test now automatically appear on the Jenkins test results page. You don't even need to keep your testing server running — all the necessary files are stored right inside the Jenkins build folder.
+
+<!--more-->
+
+The test results page should look like this with our plugin enabled:
+
+![The updated test results page](/testcafe/images/jenkins-plugin/test-results-page.png)
+
+## Test case: the disappearing button
+
+Let's say the 'Add to Cart' button disappears on the mobile version of your website.
+
+![The 'Add To Cart' button is absent from the mobile version of the website](/testcafe/images/jenkins-plugin/button-example.png)
+
+TestCafe supports several mobile device emulation methods. You can:
+
+* [Resize the browser window](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html#resize-window) during testing
+* Run the test through a [3rd party cloud testing service](https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#browsers-in-cloud-testing-services)
+* Take advantage of [Chromium's built-in emulation mode](https://devexpress.github.io/testcafe/documentation/guides/concepts/browsers.html#use-chromium-device-emulation)
+
+The third option offers greater speed and stability. We describe it going forward and recommend that you use it if possible.
+
+First, let's write a simple test that checks if the 'Add to Cart' button exists:
+
+```js
+test('`Add To Cart` button should exist', async t => {
+    await t
+        .click(Selector('button').withText('Add To Cart'));
+});
+```
+
+Now let's configure Jenkins to launch this test every time we build our project.
+
+## Basic setup
+
+### Step 1. Take care of the prerequisites
+
+TestCafe requires a working Node.js installation to run. If your testing environment does not contain node.js, [this Jenkins plugin](https://plugins.jenkins.io/nodejs/) can install it for you. Make sure you have Chromium too: TestCafe cannot function without a browser.
+
+### Step 2. Install the TestCafe Jenkins plugin
+
+To install the TestCafe Jenkins plugin, press the "Manage Plugins" link on the Manage Jenkins page, select the "Available" tab, and enter "testcafe" into the search field. Check the box next to our plugin and press the 'Install without restart' button below it.
```

Comment:
I noticed that you say "press" multiple times (instead of "click"). Is this because you are talking about a mobile device? (Just checking.)

Reply:
See https://github.com/DevExpress/testcafe/pull/5329#discussion_r458429445